test_util/cluster_api.py

```diff
@@ -274,6 +274,7 @@ def get_test_app(self, custom_port=False):
             'cpus': 0.1,
             'mem': 32,
             'instances': 1,
+            'user': 'nobody',
```

Comment:
I see. Well, that default user could be different depending on downstream config, so we can't necessarily hardcode "root" as the expected default user.
Maybe I'll just disable this test in our downstream and clone a version that is downstream-config-aware.